ts/Core/Series/Series.ts

```diff
@@ -3352,8 +3352,7 @@ class Series {
             hasRendered = series.hasRendered,
             chartSeriesGroup = chart.seriesGroup,
             inverted = chart.inverted;
-        // Animation doesn't work in IE8 quirks when the group div is
-        // hidden, and looks bad in other oldIE
+
         let animDuration = (!series.finishedAnimating && chart.renderer.isSVG) ?
```

Comment:
The `chart.renderer.isSVG` condition can be removed here, ref the removed comment.